metadata.rb

```diff
@@ -10,6 +10,7 @@
 
 depends          'apt'
 depends          'certificate'
+depends          'cron', '~> 7.0.20'
```

Comment:
Please don't set the version as we'll run into issues with having to bump this on the base cookbook. I'd rather we just do it only at the base cookbook level.

Reply:
Updated the metadata to remove the version